lib/components/narrative/default/default-itinerary.js

```diff
@@ -131,13 +131,16 @@ const ITINERARY_ATTRIBUTES = [
     id: 'cost',
     order: 2,
     render: (itinerary, options, defaultFareKey = 'regular') => {
+      const fareInCents = getTotalFare(itinerary, options.configCosts, defaultFareKey)
+      const fare = fareInCents === null ? null : fareInCents / 100
+      if (fare === null) return <FormattedMessage id="common.itineraryDescriptions.noTransitFareProvided" />
       return (
         <FormattedNumber
           currency={options.currency}
           currencyDisplay='narrowSymbol'
           style='currency'
           value={
-            getTotalFare(itinerary, options.configCosts, defaultFareKey) / 100
+            fareInCents / 100
```

Comment:
Why not use `fare` here (it is not null)?

Reply:
Good point, updated in https://github.com/opentripplanner/otp-react-redux/pull/565/commits/dbd61ab2b015f36cba00f77eb716685e0c62b8e2